packages/vuetify/src/components/VIcon/VIcon.ts

```diff
@@ -182,7 +176,7 @@ const VIcon = mixins(
     const icon = this.getIcon()
 
     if (typeof icon === 'string') {
-      if (icon.indexOf('svg-') === 0) {
+      if (icon[0] === 'M' && icon[icon.length - 1] === 'Z') {
```

Comment:
Any reason we don't just check for `this.$vuetify.icon.iconfont` here? there is an service for the iconfont but I haven't found any usage for it in the codebase? what is currently the point of setting the `iconfont` in the Vuetify constructor when it doesn't get used anywhere ?

Reply:
https://github.com/vuetifyjs/vuetify/blob/2d52dac4ebd02efaddc5289a0d29b09a6a1bd003/packages/vuetify/src/services/icons/index.ts#L14-L21

It just sets what the other components use by default, you can use any type of icon no matter what it's set to.